src/stats/LatencyStats.java

```diff
@@ -0,0 +1,103 @@
+// This file is part of OpenTSDB.
+// Copyright (C) 2015  The OpenTSDB Authors.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 2.1 of the License, or (at your
+// option) any later version.  This program is distributed in the hope that it
+// will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+// of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser
+// General Public License for more details.  You should have received a copy
+// of the GNU Lesser General Public License along with this program.  If not,
+// see <http://www.gnu.org/licenses/>.
+package net.opentsdb.stats;
+
+import com.stumbleupon.async.Deferred;
+import net.opentsdb.utils.Config;
+import net.opentsdb.utils.PluginLoader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Manages latency stats plugins for given measurement points. Supports differing plugin by measurement point as well as
+ * specifying a global default via config. If no config exists then this will return an {@link Histogram}. Ensures each 
+ * returned instance is a singleton.
+ */
+public class LatencyStats {
+
+  /** Logging */
+  private static final Logger LOG = LoggerFactory.getLogger(LatencyStats.class);
+
+  /** Created instances */
+  private static ConcurrentHashMap<String, LatencyStatsPlugin> instances = new ConcurrentHashMap<String, LatencyStatsPlugin>();
+
+  /**
+   * Get the latency stats plugin for a given measurement point.
+   * @param config The configuration for this TSDB instance
+   * @param instanceName The named measurement point
+   * @param metricName The name of the metric to emit aggregations to
+   */
+  public static LatencyStatsPlugin getInstance(Config config, String instanceName, String metricName) {
```

Comment:
Probably better to just pass in the TSDB object so we can pull out the config and possibly make additional calls in the future.
Also need Finals and TSDB style non-camel case param names.